guard/src/commands/validate/common.rs

```diff
@@ -661,13 +606,12 @@ where
                                     } else {
                                         "was int"
                                     },
-                                IsFloat =>
+                                _ =>
```

Comment:
Is this change expected?

Looks like `_` is the default arm, but we have messages that are related to `cmp` being a float?

Reply:
CmpOperator doesnt have an IsFloat variant